uint8 slave_id
uint8 imu_send_data_flag
spinal/ServoInfo[] servos
